pkgs/applications/science/logic/isabelle/default.nix

```diff
@@ -57,6 +63,12 @@ in stdenv.mkDerivation rec {
     sourceRoot=${dirname}
```

Comment:
This should use sourceRoot directly if possible and lib.optionalString

Reply:
what do you mean by that?